onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -502,6 +505,23 @@ def _where_input_wrangler(
         matcher=lambda sample: sample.kwargs.get("rounding_mode") is not None,
         reason="rounding_mode is not yet supported",
     ),
+    skip(
+        "min",  # aten_mean
+        matcher=lambda sample: len(sample.args) > 0,
+        reason="only support one tensor as input",
```

Comment:
```suggestion
        reason="this ATen overload only supports one tensor as input by design",
```

for clarity so that we know it's by design. here and below